mpas_analysis/config.default

```diff
@@ -696,8 +696,8 @@ movingAveragePoints = 1
 usePostprocessingScript = False
 
 # Region names for basin MOC calculation.
-# Supported options are Atlantic and IndoPacific
-regionNames = ['Atlantic', 'IndoPacific']
+# Supported options are Atlantic, AtlanticMed, Indian, Pacific and IndoPacific
+regionNames = ['Atlantic', 'AtlanticMed', 'IndoPacific']
```

Comment:
@milenaveneziani, just so it's clear, this list still only has some of the regions, not all.

Reply:
I am not too attached to the separate Indian and Pacific MOC's, at least at the moment. So I think this is fine. And 'Global' is not included in here because that is done separately, right?

Reply:
Right, I don't think 'Global' is optional at the moment. You always get it.

Reply:
And that's a good thing.
The 3 regions included here is all we'll need for most cases.